join.sh

```diff
@@ -70,3 +70,5 @@ docker run -t --rm \
         join \
             --hostname $(hostname) \
             --config-path /target$(realpath $1)
+
+touch /var/lib/prom.done
```

Comment:
The issue I see w/ a log file is that something could be logged to say promenade didn't work. But that could happen early enough that promenade wants to be run again on next boot to retry the join.